ansible/library/dual_tor_facts.py

```diff
@@ -25,10 +25,10 @@ def parse_tor_position(self):
         '''
         Determines the position ('U' for upper and 'L' for lower) of the ToR.
 
-        The upper ToR is always the first ToR alphabetically by hostname
+        The upper ToR is always the first ToR listed in the testbed file
         '''
-        upper_tor, lower_tor = sorted(self.testbed_facts['duts'])
-        self.dual_tor_facts['positions'] = {'upper': upper_tor, 'lower': lower_tor}
+        upper_tor, lower_tor = self.testbed_facts['duts']
```

Comment:
Looks like LGTM found out that these vars `upper_tor, lower_tor` are no longer used/needed.